kernel/base/src/main/java/com/twosigma/beakerx/widgets/selections/SelectionWidget.java

```diff
@@ -54,4 +54,12 @@ public void setSize(Integer size) {
     sendUpdate(SIZE, size);
   }
 
+  public String getModelModuleValue(){
+    return "@jupyter-widgets/controls";
+  }
+
+  public String getViewModuleValue(){
+    return "@jupyter-widgets/controls";
```

Comment:
Is this change necessary while in other examples we are using `jupyter-js-widgets` ?